### PR TITLE
Centrifuge App: Liquidity pools investments

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

Adds liquidity pools investments

### Approvals

- [ ] Dev

### CODE REVIEW

File: centrifuge-app/src/pages/IssuerPool/Investors/LiquidityPools.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      )
+      return results.some((result) => result.status === 'fulfilled' && result.value.length > 0)
+    },
+    {
+      enabled: !!managerAddress,
+      staleTime: Infinity,
+    }
+  )
+  const { execute, isLoading } = useCentrifugeTransaction(
+    'Update investor',
```
Comment (sophialittlejohn) [inline centrifuge-app/src/pages/IssuerPool/Investors/LiquidityPools.tsx:76]: How about 'Enable \<network\>'?

File: centrifuge-js/src/modules/liquidityPools.ts [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+            router: '0x205b09C1C11A0a86cbc1066CC7B54C7952427439',
+          },
+        ]
+      })
+    )
+  }
+
+  async function getManagerFromRouter(args: [router: string], options?: EvmQueryOptions) {
+    const [router] = args
+    const gatewayAddress = await contract(router, ABI.Router, options).gateway()
```
Comment (sophialittlejohn) [inline centrifuge-js/src/modules/liquidityPools.ts:124]: what does this gateway do/facilitate?

**Review by sophialittlejohn — COMMENTED** [review]
Nice job!! There are still few logs, comments and todos left in the code. 

Also, which USDC do I need to test investing? I have some goerli USDC ([tx](https://goerli.etherscan.io/tx/0xd5f277ae14b85da66a914780cce9bbadc50623430aa6890ea1108c59469ea488)) but the balance in the invest input still said 0

**Comment by onnovisser** [comment]
@sophialittlejohn  this is the Goerli USDC used `0xd35CCeEAD182dcee0F148EbaC9447DA2c4D449c4`

File: centrifuge-js/src/modules/liquidityPools.ts (hunk 2 above)
Comment (onnovisser) [reply centrifuge-js/src/modules/liquidityPools.ts:124]: I think it manages the routers and handles the incoming/outgoing messages

**Review by sophialittlejohn — APPROVED** [review]

**Pull request merged**